### ISSUE TITLE
set ngmlr parameters

### ISSUE THREAD

**Issue by colindaven** [MHH-RCUG/Wochenende#72]
--min-identity 0.90 (0.85 currently used !)
--min-residues 0.60
```
    -i <0-1>,  --min-identity <0-1>
        Alignments with an identity lower than this threshold will be discarded [0.65]
    -R <int/float>,  --min-residues <int/float>
        Alignments containing less than <int> or (<float> * read length) residues will be discarded [0.25]



# Existing code (--min-residues is still default)
    ngmlrMinIdentity = (
        0.85  # Aligner ngmlr only: minimum identity (fraction) of read to reference
    )
```

**Comment by colindaven** [comment]
--min-identity of 0.95 was trialled for long read mock community metagenomes in August 2020. Too high, very few reads were aligned. Stick to 0.85

--min-residues - maybe 0.50- 0.75 sounds more interesting ?

**Comment by colindaven** [comment]
Implemented for 0.70 
Needs rigourous testing with mock community.
